docs/discover/try-esql.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[try-esql]]
+== Try ES|QL
+
+preview::[]
+
+The Elasticsearch Query Language, ES|QL, has been created to make exploring your data easier without leaving the Discover application. 
+
+In the tutorial below we'll use the {kib} Sample Data Logs in Discover and Lens suggestions to explore the data and visualization. 
+
+[float]
+[[prerequisite]]
+=== Prerequisite 
+
+To be able to select **Try {esql}** from the Data views menu the `discover:enableESQL` setting must be enabled from **Stack Management > Advanced Settings**. It is enabled by default. 
+
+[float]
+[[tutorial-try-esql]]
+=== Trying ES|QL 
+
+Learn how to build queries to pull information out of your data and create visualizations using {esql}. 
+
+To get the sample data into Discover:
+
+. On the home page, click **Try sample data**.
+. Click **Other sample data sets**.
+. On the Sample web logs card, click **Add data**.
+. Open the main menu and select *Discover*.
+. From the Data views menu, select *Try ES|QL*.
++
+[role="screenshot"]
+image:images/try-esql.png[An image of the Discover UI where users can try the new ES|QL feature]
+
+Let's say we want to find out what machine operating system users have and how much ram is on their machine.  
+
+[NOTE]
+====
+{esql} keywords are not case sensitive. 
+====
+
+. Set the time range to **Last 7 days**.
+. Expand image:images/expand-icon-2.png[An image of the expand icon] the query bar.
+. Put each processing command on a new line for better readability.
+. Copy the query below:
++
+[source,esql]
+----
+from kibana_sample_data_logs 
+| keep machine.os, machine.ram
+----
```

Comment:
```suggestion
----
FROM kibana_sample_data_logs 
| KEEP machine.os, machine.ram
----
```
We're using capitalized keywords for ES|QL in the ES docs. Probably good to be consistent.